plugins/stack-overflow/src/alpha.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { configApiRef, createApiFactory } from '@backstage/core-plugin-api';
+import {
+  createApiExtension,
+  createPlugin,
+} from '@backstage/frontend-plugin-api';
+import { createSearchResultListItemExtension } from '@backstage/plugin-search-react/alpha';
+import { StackOverflowClient, stackOverflowApiRef } from './api';
+
+/** @alpha */
+const StackOverflowApi = createApiExtension({
+  factory: createApiFactory({
+    api: stackOverflowApiRef,
+    deps: { configApi: configApiRef },
+    factory: ({ configApi }) => StackOverflowClient.fromConfig(configApi),
+  }),
+});
+
+/** @alpha */
+const StackOverflowSearchResultListItem = createSearchResultListItemExtension({
```

Comment:
did we say that we wanted to still export these or should we hold off on that? @Rugvip

Reply:
I'd say let's not export for now, they're kinda useless anyway so we'd need to figure out what to do with them

Reply:
... and tbh I think I prefer making them available on the plugin instance somehow instead if we're gonna do that, feeling it might get really messy to have a bunch of separate extension exports